src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer: do work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer: stop work");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It should indicate that the Bulldozer has stopped working, for example: 'Bulldozer stopped its work.'